financial-templates-lib/truffle-config.js

```diff
@@ -0,0 +1,9 @@
+const path = require("path");
```

Comment:
Since this `config` is repeated for all of the packages, is it worth writing it in a single file ("truffle-config-template.js") and importing that from all the packages?